File: server/database/conn.go
package database

import (
	"context"
	"fmt"
	"os"

	"go.mongodb.org/mongo-driver/mongo"
	"go.mongodb.org/mongo-driver/mongo/options"
)

var SaveData *mongo.Collection

func ConnectDB() error {
	 db :=  os.Getenv("DB")
	clientOption := options.Client().ApplyURI(db)
	client, err := mongo.Connect(context.TODO(), clientOption)

	if err != nil {
		return err
	}
	fmt.Println("Database connected")
	SaveData = client.Database("Next").Collection("e-commerce")

	return nil
}

File: server2/configDB/conn.js
const mongoose = require('mongoose')
require('dotenv').config();

// Connect to MongoDB
const connectdb = () => {
    mongoose.connect(process.env.MONGO_DB)
        .then(() => console.log('Connected to MongoDB'))
        .catch((err) => console.error('Error connecting to MongoDB:', err));
}

module.exports = connectdb